scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -29,16 +29,26 @@ object JSFacade {
     }.asInstanceOf[JSLong]
   }
 
+  private[this] def toNode(t: Any): js.Any = t match {
+    case t: Tree => toNode(t)
+    case tt: Seq[_] => tt.map(toNode).toJSArray
+    case t: Option[_] => t.map(toNode).orUndefined
+    case _ => ()
+  }
+
   private[this] def toNode(t: Tree): js.Dynamic = {
     val base = js.Dynamic.literal(
       "type" -> t.productPrefix,
-      "children" -> t.children.map(toNode).toJSArray,
```

Comment:
Ah, got it.
It would be an easy addition, but I don't have a use case in mind for it atm.

I would keep it out of the scope of this PR and add it on demand in case it's deemed useful.